.github/workflows/generate-feature-list.yml

```diff
@@ -33,18 +39,20 @@ jobs:
             echo "INPUT_SPREADSHEET_URI=https://docs.google.com/spreadsheets/d/e/2PACX-1vQwzrUSKfuSRcpkp7sJTw1cSB63s4HCjYLJeGPWECsvqn222hjaaONQlN4X8auKvlaB0es3BqV5rQyz/pub?gid=829069645&single=true&output=csv" >> $GITHUB_ENV
           fi
 
-      # Step 4: Fetch and process the pricing list by directly running the Go file
-      - name: Fetch and process pricing list
+      # Step 4: Fetch and process the feature list by directly running the Go file
+      - name: Fetch and process feature list
         run: |
-          go run ./.github/actions/pricing-list-action/main.go
+          go run ./.github/actions/feature-list-action/main.go
 
       - name: Commit changes
         uses: stefanzweifel/git-auto-commit-action@v5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} 
```

Comment:
FYI you can utilize `RELEASEDRAFTER_PAT` or `GH_ACCESS_TOKEN` which belong to l5io